### ISSUE TITLE
 Some questions about error handling on runc 

### ISSUE THREAD

**Issue by payall4u** [opencontainers/runtime-spec#1061]
runc will get `container <container-id> does not exist`
crun will get `error opening file 'container-status': No such file or directory\n: unknown"`

**Comment by giuseppe** [comment]
maybe we could at least agree on a few exit codes so that container engines don't have to parse the error string.

**Comment by cyphar** [comment]
I agree that standardising some basic exit codes would be a good idea.

**Comment by utam0k** [comment]
@giuseppe @cyphar @AkihiroSuda 
What is the status of this issue? I have interested in this. This is a good idea!

**Comment by utam0k** [comment]
How about simply adding 255 to the error code if there is an error inside OCI Runtime? It would be easier for the client to handle.
